src/hooks/my/useItemsPerPage.jsx

```diff
@@ -1,27 +1,36 @@
 import { useState, useEffect } from 'react';
 
-function useItemsPerPage() {
-    const [itemsPerPage, setItemsPerPage] = useState(16);
+// 화면 크기를 나타내는 상수 객체
+const SIZES = {
+    mobile: 768,
+    tablet: 1280,
+};
+
+// 화면 크기에 따라 보여줄 아이템 개수를 동적으로 설정
+function useItemsPerPage(items = { mobile: 6, tablet: 8, desktop: 16 }) {
+    const [itemsPerPage, setItemsPerPage] = useState(items.desktop);
```

Comment:
```suggestion
    const [itemsPerPage, setItemsPerPage] = useState(getItemsPerPage);
```
초기 설정을 현재 화면 크기에 맞게 설정하도록 수정해서 불필요한 초기 렌더링을 방지할 수 있을 것 같아요